src/browser-bookmarks/import-bookmarks.js

```diff
@@ -0,0 +1,41 @@
+import db from '../pouchdb'
+import { updatePageSearchIndex } from '../search/find-pages'
+import { getVisitItemsForHistoryItems, transformToPageDoc, transformToVisitDoc,
+   convertHistoryToPagesAndVisits } from '../browser-history/import-history'
+import { isWorthRemembering, generatePageDocId, generateVisitDocId,
+  visitKeyPrefix, convertVisitDocId } from '../activity-logger'
+
+    function onRejected(error) {
+      console.log(`An error: ${error}`);
+    }
+
+    function getBookmarkItems() {
+      var bookmarkTree = browser.bookmarks.getRecent(100000);
```

Comment:
`bookmarkTree` is not the right name anymore. It does not really need a name though, you can just return the whole promise chain directly.